src/aind_data_transfer_service/models.py

```diff
@@ -32,6 +32,22 @@ class AirflowDagRunsResponse(BaseModel):
     total_entries: int
 
 
+class AirflowDagRunsRequestParameters(BaseModel):
+    """Model for parameters when requesting info from dag_runs endpoint"""
+
+    limit: int = 25
+    offset: int = 0
+    start_date_gte: str = (datetime.utcnow() - timedelta(days=14)).strftime(
```

Comment:
We can probably remove this if we are paginating the query

Reply:
Removed `start_date_gte` filter